src/components/common/Footer/index.tsx

```diff
@@ -65,11 +57,12 @@ const Footer = (): ReactElement | null => {
             </li>
             <li>
               <Link href={AppRoutes.cookie} passHref>
-                <MUILink>Cookie Policy</MUILink>
+                <MUILink>Cookie policy</MUILink>
               </Link>
-              &nbsp;&mdash;&nbsp;
-              <Link href={`${router.pathname}#cookie-preferences`} passHref>
-                <MUILink onClick={onCookieClick}>Preferences</MUILink>
```

Comment:
With this removed, its not possible to update the cookie preferences anymore unless going to the non-safe settings page. Is there a reason to move the cookie preferences there or could we still keep it as a popup like it was?

Reply:
You can still click on Settings in the footer from any page where the footer is displayed, and it will open cookie prefs.